### ISSUE TITLE
can't find McDonald's app in sesame search

### ISSUE THREAD

**Issue by canojim** [BranchMetrics/sesame_issues#165]
https://play.google.com/store/apps/details?id=com.mcdonalds.mobileapp

The app can be found in launcher but not sesame

**Comment by philwall2** [comment]
Good catch. We've had a few reports of this and looked into it but aren't sure why it doesn't show. It's possible it's our fault but also their app seems weird. We're using pretty standard methods for detecting and opening apps. Not sure why McDonald's works differently. 

TBH we took a look and didn't see a cause, so we don't want to put too much more effort into this edge case.
